### ISSUE TITLE
Extension configuration not found

### ISSUE THREAD

**Issue by noll1** [aimeos/aimeos-typo3#77]
I've installed aimeos in a fresh Typo3 environment.
After I activate the extension following error occured:
![bug](https://user-images.githubusercontent.com/42864489/47289406-dbbdf600-d5fa-11e8-9323-f07e78039583.png)
Also it is not possible to run the update script. Get the same error.

**Comment by aimeos** [comment]
Might be a problem of the new TYPO3 9.5.0 release. Is there any useful information if you click on the "online" link?

**Comment by noll1** [comment]
I think I found the problem.
I've installed the extension over the TER in the backend.
Today I tried it with the .zip archive from https://extensions.typo3.org/extension/aimeos .
Then I compared these two packages from Website and TER.
In my case the file "ext_conf_template.txt" was missing in the TER package and in the .zip archive the complete aimeos library (EXT:aimeos/Resources/Libraries & EXT:aimeos/Resources/Private/Extensions) was missing.
After I merged these two folders it was possible to install the extension.
Maybe you forgot some files during deployment process? 
Because when I download the version 18.7.3 the aimeos library is in it and the .zip archive is 7MB greater than 18.10.0.
